.travis.yml

```diff
@@ -2,25 +2,22 @@ language: php
 dist: trusty
 
 php:
-  - 7.1
   - 7.2
```

Comment:
Shall we really keep supporting this version as well?

Reply:
7.2 is what symfony 5 supports, and currently (this) library does so too, so why would we drop it?

Reply:
The version 7.2 is not actively supported anymore. The version 7.3 won't be actively supported in 4 months.

For me, not supporting older versions means less maintenance but it's also an statement. The last version of PHPUnit, for example, required at least PHP 7.3, and if you can't upgrade your PHP version you're stuck with an older version of PHPUnit.

By keeping the support we retrain ourselves of using new features of PHP, we're stuck to older versions of other packages, and we're supporting people/companies to not upgrade their PHP version.

Reply:
Ok, I will drop the support and tag a new major version.